pyproject.toml

```
@@ -0,0 +1,37 @@
+[tool.poetry]
+name = "seedcase examples"
```

```suggestion
name = "examples"
```

To match the name of the folder.